atom/browser/native_window_mac.mm

```diff
@@ -1682,12 +1684,30 @@ static bool FromV8(v8::Isolate* isolate, v8::Handle<v8::Value> val,
   }
 }
 
+void NativeWindowMac::SetRenderWidgetHostOpaque(bool opaque) {
+  if (web_contents()) {
+    auto render_view_host = web_contents()->GetRenderViewHost();
+    if (render_view_host) {
+      content::RenderWidgetHostImpl* impl = content::RenderWidgetHostImpl::FromID(
+          render_view_host->GetProcess()->GetID(),
+          render_view_host->GetRoutingID());
+      if (impl)
+        impl->SetBackgroundOpaque(opaque);
+    }
+  }
+}
```

Comment:
This is a pretty egregious [demeter violation](https://hackernoon.com/object-oriented-tricks-2-law-of-demeter-4ecc9becad85) but I'm not sure what the Right Thing is here since some of these conditionals are for upstream classes out of our control :thinking:

Reply:
Yeah, I tried to collapse it as much as I could but everything is just consecutive potentially nullptr values 🤔

Reply:
I inverted the logic to collapse it a bit 👍

Reply:
Still like 4 steps of separation though so idk 🤔